packages/proxy/src/proxy.ts

```diff
@@ -276,10 +300,17 @@ export async function proxyV1({
     bodyData.temperature !== 0 &&
     (bodyData.seed === undefined || bodyData.seed === null);
 
-  const useCache =
+  const readFromCache =
```

Comment:
should we check cacheMaxAge right here? may be simpler if we just treat it as equivalent to no-cache

Reply:
yeah we can check it for the max-age=0 scenario here but otherwise we need to reach into the cache to see if it meets the requested max age